pants-plugins/internal_plugins/test_lockfile_fixtures/BUILD

```diff
@@ -0,0 +1,4 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+python_sources(dependencies=["src/python/pants/testutil"])
```

Comment:
I'm not sure why this wasn't inferred, but: 1) would add a TODO indicating that, 2) would make it more specific by pointing to the actual file used `src/python/pants/testutil/lockfile_fixture.py`.

Reply:
Done.